airbyte-integrations/connectors/destination-oracle-strict-encrypt/src/test-integration/java/io/airbyte/integrations/destination/oracle_strict_encrypt/OracleStrictEncryptDestinationAcceptanceTest.java

```diff
@@ -151,15 +150,13 @@ protected void setup(final TestDestinationEnv testEnv, final HashSet<String> TES
     db.start();
 
     config = getConfig(db);
+    final DSLContext dslContext = getDslContext(config);
```

Comment:
is it OK to not close the DSLContext?

Reply:
:doh: good catch

Reply:
actually it looks like the underlying code has changed the closing of this object doesn't work. I believe the cleanup all happens in the tear down methods for this test